flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/flink/FlinkContainerTestEnvironment.java

```diff
@@ -68,6 +71,7 @@ public FlinkContainerTestEnvironment(
         config.set(HEARTBEAT_INTERVAL, 1000L);
         config.set(HEARTBEAT_TIMEOUT, 5000L);
         config.set(SLOT_REQUEST_TIMEOUT, 10000L);
+        config.set(METRIC_FETCHER_UPDATE_INTERVAL, 1000L);
```

Comment:
We can use constant to maintain these magic number, these are used for tests with default constant value IIUC.
btw, Could connector developer pass custom Configuration?

Reply:
fixed